src/modules/networks/networks-store.background.ts

```diff
@@ -1,16 +1,99 @@
+import { client } from 'defi-sdk';
+import type { WalletRecord } from 'src/shared/types/WalletRecord';
 import { chainConfigStore } from '../ethereum/chains/ChainConfigStore';
+import { configureBackgroundTestClient } from '../defi-sdk/background';
+import type { ChainId } from '../ethereum/transactions/ChainId';
 import { NetworksStore } from './networks-store';
+import type { NetworkConfig } from './NetworkConfig';
+import type { Chain } from './Chain';
 
-export const networksStore = new NetworksStore(
+export const mainNetworksStore = new NetworksStore(
   { networks: null },
   {
     getEthereumChainConfigs: async () => {
       await chainConfigStore.ready();
       return chainConfigStore.getState().ethereumChainConfigs;
     },
+    client,
+    testnetMode: false,
   }
 );
 
+export const testenvNetworksStore = new NetworksStore(
+  { networks: null },
+  {
+    getEthereumChainConfigs: async () => {
+      await chainConfigStore.ready();
```

Comment:
Do we need to set client for chainConfigStore here to testnet?

Reply:
Not really since they share same config store, but thanks for pointing out still, I just realized I'm not setting it initially in `Wallet`